examples/time_frequency/plot_single_trial_spectra.py

```diff
@@ -35,48 +35,21 @@
 include = []
 raw.info['bads'] += ['MEG 2443']  # bads
 
-# picks MEG gradiometers
-picks = mne.pick_types(raw.info, meg='grad', eeg=False, eog=True,
-                       stim=False, include=include, exclude='bads')
-
-epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks, proj=True,
-                    baseline=(None, 0), reject=dict(grad=4000e-13, eog=150e-6))
-
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
+                    baseline=(None, 0),
+                    reject=dict(grad=4000e-13, eog=150e-6))
 
 n_fft = 256  # the FFT size. Ideally a power of 2
-psds, freqs = compute_epochs_psd(epochs, fmin=2, fmax=200, n_fft=n_fft,
-                                 n_jobs=2)
-
-# average psds and save psds from first trial separately
-average_psds = psds.mean(0)
-average_psds = 10 * np.log10(average_psds)  # transform into dB
-some_psds = 10 * np.log10(psds[12])
-
-
-fig, (ax1, ax2) = plt.subplots(1, 2, sharex=True, sharey=True, figsize=(10, 5))
 
-fig.suptitle('Single trial power', fontsize=12)
+# Let's first check out all channel types by averaging across epochs.
+epochs.plot_psds(plot_kind=1, fmin=2, fmax=200, n_fft=n_fft, n_jobs=2)
```

Comment:
would be more consistent with how we handle things + more explicit to have different name here instead of a type/mode argument, why not epochs.plot_psds_image/raster ?